<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>10</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FaceAI Targets</string>
  </property>
  <property name="windowIcon">
   <iconset theme="FaceAI" resource="../images/images.qrc">
    <normaloff>:/newPrefix/FaceAI-Logo.png</normaloff>
    <normalon>:/newPrefix/FaceAI-Logo.png</normalon>
    <disabledoff>:/newPrefix/FaceAI-Logo.png</disabledoff>
    <disabledon>:/newPrefix/FaceAI-Logo.png</disabledon>
    <activeoff>:/newPrefix/FaceAI-Logo.png</activeoff>
    <activeon>:/newPrefix/FaceAI-Logo.png</activeon>:/newPrefix/FaceAI-Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/Background.png);
border:none;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblLogo">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>100</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>200</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../images/FaceAI-Logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="cmdbtnGoBack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-size: cover;
background-repeat: no-repeat;
background-color: rgb(127, 0, 226);
border-radius: 10px;
background:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../images/images.qrc">
            <normaloff>:/newPrefix/icon-back2.png</normaloff>:/newPrefix/icon-back2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>200</number>
        </property>
        <property name="topMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>200</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stkwdtSelectPhotos">
          <property name="styleSheet">
           <string notr="true">background: transparent;
margin: 10px;
</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="wdtSinglePhoto">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnSinglePhoto">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>400</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:transparent;
border:0px;
border-image:url(:/newPrefix/Group 67.png);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wdtMultiPhotos">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="btnMultiPhoto">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border:none;
image: url(:/newPrefix/icon-folder.png);
background:transparent;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../images/images.qrc">
                  <normaloff>:/newPrefix/icon-folder.png</normaloff>:/newPrefix/icon-folder.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <item alignment="Qt::AlignLeft|Qt::AlignBottom">
                 <widget class="QLabel" name="label_10">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>400</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
background:transparent;</string>
                  </property>
                  <property name="text">
                   <string>Select target photos</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lblMultiResult">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgba(255, 255, 255, 0.4);
background:transparent;</string>
                  </property>
                  <property name="text">
                   <string>Raster image formats are accepted</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="entireFolder">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btnEntireFolder">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border:none;
background:transparent;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../images/images.qrc">
                  <normaloff>:/newPrefix/icon-folder.png</normaloff>:/newPrefix/icon-folder.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>120</width>
                  <height>120</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <item alignment="Qt::AlignLeft|Qt::AlignBottom">
                 <widget class="QLabel" name="lblEntireFolder">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);
background:transparent;</string>
                  </property>
                  <property name="text">
                   <string>Select target folder</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignLeft|Qt::AlignTop">
                 <widget class="QLabel" name="lblEntireFolderResult">
                  <property name="styleSheet">
                   <string notr="true">color: rgba(255, 255, 255, 0.4);
background:transparent;</string>
                  </property>
                  <property name="text">
                   <string>Raster image formats are accepted</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="oldCasePhoto">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="lblOldCaseResult">
              <property name="minimumSize">
               <size>
                <width>700</width>
                <height>200</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background:transparent;</string>
              </property>
              <property name="text">
               <string>Click on the &quot;Start probe&quot; button below to continue the further process.</string>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="rdobtnSinglePhoto">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>17</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>17</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton:unchecked {
color: rgba(255, 255, 255,0.6);
}

QRadioButton:checked {
color: rgba(255, 255, 255,0.6);
}
QRadioButton{
background:transparent;
}

</string>
          </property>
          <property name="text">
           <string>Single photo  </string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdobtnMultiPhoto">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>17</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>17</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton:unchecked {
color: rgba(255, 255, 255,0.6);
}

QRadioButton:checked {
color: rgba(255, 255, 255,0.6);
}
QRadioButton{
background:transparent;
}

</string>
          </property>
          <property name="text">
           <string>Multiple photo  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdobtnEntireFolder">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>17</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>17</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton:unchecked {
color: rgba(255, 255, 255,0.6);
}

QRadioButton:checked {
color: rgba(255, 255, 255,0.6);
}


QRadioButton{
background:transparent;
}</string>
          </property>
          <property name="text">
           <string>Entire folder of photo  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdobtnOldCasePhoto">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>17</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>17</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QRadioButton:unchecked {
color: rgba(255, 255, 255,0.6);
}

QRadioButton:checked {
color: rgba(255, 255, 255,0.6);
}


QRadioButton{
background:transparent;
}</string>
          </property>
          <property name="text">
           <string>Old case photo  </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="btnStartProbe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/start_prob_button_img.png</normaloff>../images/start_prob_button_img.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>250</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QPushButton" name="btnReturnHome">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

border:none;
background:transparent;</string>
      </property>
      <property name="text">
       <string>Return to Home</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
